README.md

```diff
@@ -776,6 +776,60 @@ wait_agent(3, question, ScKeynodes[QuestionStatus.QUESTION_FINISHED])
 question, is_successfully = execute_agent(**kwargs, wait_time=3)  # ScAddr(...), bool
 ```
 
+### Prepare, call and execute action
+
+Function that allow to create action and call later.
+
+Prepare action function creates action and adds arguments and concepts.
+
+```python
+def prepare_action(
+        arguments: Dict[ScAddr, IsDynamic],
+        concepts: List[Idtf],
+) -> ScAddr: ...
+```
+
+Then call or execute action like `call_agent` or `execute_agent`.
+Action call functions don't return action_node because it's parameter to them.
+
+```python
+def call_action(
```

Comment:
Are there cases when you need to call this functions separately?

Reply:
No, because one of them is part of another